docs/modules/ROOT/pages/api/access.adoc

```diff
@@ -0,0 +1,477 @@
+:github-icon: pass:[<svg class="icon"><use href="#github-icon"/></svg>]
+:AccessControl: xref:AccessControl[AccessControl]
+:Ownable: xref:Ownable[Ownable]
+:src5: https://github.com/starknet-io/SNIPs/blob/main/SNIPS/snip-5.md[SRC5]
+:inner-src5: xref:api/introspection.adoc#ISRC5[SRC5 ID]
+:_set_role_admin: xref:#AccessControl-_set_role_admin[_set_role_admin]
+
+= Access Control
+
+[.readme-notice]
+NOTE: This document is better viewed at https://docs.openzeppelin.com/contracts/api/access
+
+This directory provides ways to restrict who can access the functions of a contract or when they can do it.
+
+- {Ownable} is a simple mechanism with a single "owner" role that can be assigned to a single account.
+This mechanism can be useful in simple scenarios, but fine grained access needs are likely to outgrow it.
+- {AccessControl} provides a general role based access control mechanism. Multiple hierarchical roles can be created and
+assigned each to multiple accounts.
+
+== Authorization
+
+[.contract]
+[[Ownable]]
+=== `++Ownable++` link:https://github.com/OpenZeppelin/cairo-contracts/blob/release-0.7.0/src/access/ownable/ownable.cairo[{github-icon},role=heading-link]
+
+```javascript
+use openzeppelin::access::ownable::Ownable;
+```
+
+`Ownable` provides a basic access control mechanism where an account
+ (an owner) can be granted exclusive access to specific functions.
+
+This module includes the `assert_only_owner` internal to restrict a function to be used only by the owner.
+
+[.contract-index]
+.External Functions
+--
+.OwnableImpl
+
+* xref:Ownable-owner[`++owner(self)++`]
+* xref:Ownable-transfer_ownership[`++transfer_ownership(self, new_owner)++`]
+* xref:Ownable-renounce_ownership[`++renounce_ownership(self)++`]
+--
+
+[.contract-index]
+.Internal Functions
+--
+.InternalImpl
+
+* xref:Ownable-initializer[`++initializer(self, owner)++`]
+* xref:Ownable-_transfer_ownership[`++_transfer_ownership(self, new_owner)++`]
+* xref:Ownable-assert_only_owner[`++assert_only_owner(self)++`]
```

Comment:
```suggestion
* xref:Ownable-assert_only_owner[`++assert_only_owner(self)++`]
* xref:Ownable-_transfer_ownership[`++_transfer_ownership(self, new_owner)++`]
```
^So that they match the order in the contract. I'd switch their respective descriptions below as well